dotenv/mod_test.ts

```diff
@@ -20,10 +22,10 @@ import { assertType } from "../testing/types.ts";
 const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
 const testdataDir = path.resolve(moduleDir, "testdata");
 
-const testOptions = {
-  envPath: path.join(testdataDir, "./.env"),
-  defaultsPath: path.join(testdataDir, "./.env.defaults"),
-};
+const testOptions = Object.freeze({
+  envPath: path.join(testdataDir, ".env"),
+  defaultsPath: path.join(testdataDir, ".env.defaults"),
+});
```

Comment:
This is not strictly necessary, but I had to examine the entire test file to be sure that nothing was mutating these properties before I felt confident that I could rely on their values in an additional test. This change will prevent someone else from having to do the same.